### PR TITLE
docs(index.html): add getting started section

### PR DESCRIPTION
**Pull request by marcoscaceres**
CC @caribouW3

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/spec-prod/pull/69.html" title="Last updated on Jun 1, 2021, 10:03 AM UTC (b439fd1)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/spec-prod/69/e88ba2d...b439fd1.html" title="Last updated on Jun 1, 2021, 10:03 AM UTC (b439fd1)">Diff</a>

### CODE REVIEW

**Comment by marcoscaceres** [comment]
Actually, let me fix the linking here...

**Comment by marcoscaceres** [comment]
(not sure what's up with the Prettier... but that's what it outputs for me locally 🤢)

**Comment by sidvishnoi** [comment]
I was wondering if we can create a docs/README.md and move abstract and getting started there, and then data-include it in index.html
That way, we can have it rendered in both places. Also, markdown is kinda nicer to write.

**Comment by marcoscaceres** [comment]
We can do the README thing... will it still show up as the README for the repo itself?

**Comment by sidvishnoi** [comment]
> will it still show up as the README for the repo itself?

Not in repo itself, but when someone visits `docs` folder in repo. We can maybe replace our README if this ads more value?

**Comment by marcoscaceres** [comment]
Ok, so we will have two README.md? That's ok, given we have... 

Alternatively, we could just convert the respec file into markdown? What's the value in having the separate files .md files?

**Comment by sidvishnoi** [comment]
`README.md` is more of an introduction to the repo. `docs/` is actual documentation/user guide.

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+To get started, do the following:
+
+1. [Request an Echidna token](https://www.w3.org/Web/publications/register) for your spec (W3C Team Members and Chairs only!).
+1. Save the token as a "Secret" in the spec's repository's settings.
+1. Create a ".github/workflows/auto-publish.yml" file at the root of the spec's repository.
+   If you are using VS Code:
+   ```bash"
+   code .github/workflows/auto-publish.yml
+   ```
```
Comment (sidvishnoi) [inline docs/README.md:11]: I think people know how to create files 😅 
```suggestion
   If you are using VS Code:
   ```bash"
   code .github/workflows/auto-publish.yml
   ```
```

File: docs/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+         - uses: w3c/spec-prod@v2
+         with:
+           TOOLCHAIN: respec # or bikeshed
+           W3C_ECHIDNA_TOKEN: ${{ secrets.ECHIDNA_TOKEN }}
+           W3C_WG_DECISION_URL: " SEE BELOW FOR URLS! "
+           # Consider enabling link validation!
+           # VALIDATE_LINKS: true
+           # Convert Editor's Draft to Working Draft!
+           W3C_BUILD_OVERRIDE: |
+           specStatus: WD
```
Comment (sidvishnoi) [inline docs/README.md:40]: Indent is wrong here.

File: docs/options.md [hunk 3]
```diff
@@ -138,7 +138,7 @@ The automated publication workflow requires a [token](https://github.com/w3c/ech
 
 **Default:** None.
 
-## `W3C_WG_DECISION_URL`
+## `W3C_WG_DECISION_URL` {#w3c_wg_decision_url}
```
Comment (sidvishnoi) [inline docs/options.md:141]: I would prefer if we fix the ReSpec bug that's requiring this instead.

**Review by sidvishnoi — CHANGES_REQUESTED** [review]

File: docs/README.md (hunk 2 above)
Comment (sidvishnoi) [reply docs/README.md:40]:
```diff
     jobs:
       validate-and-publish:
-       name: Validate and Publish to TR
+        name: Validate and Publish to TR
```

**Comment by sidvishnoi** [comment]
I don't understand why my local spec-prod build pass with these docs but it fails here (that bad link that markup validator catches).

File: docs/README.md [hunk 4]
```diff
@@ -2,41 +2,36 @@
...
+   1. [Request an Echidna token][request-token] for your spec (W3C Team Members and Chairs only!).
+   1. [Save the token][save-token] as a "Secret" named `ECHIDNA_TOKEN` in the spec's repository's settings.
+1. Create a `.github/workflows/auto-publish.yml` file at the root of the spec's repository.
+1. In the `auto-publish.yml`, copy-paste and modify one of the [examples](examples.md) below that suits your needs. Most typical one:
 
    ```yml
-    # Inside .github/workflows/auto-publish.yml
-    name: Node CI
+   # Inside .github/workflows/auto-publish.yml
+   name: CI
```
Comment (marcoscaceres) [inline docs/README.md:13]: I think we should rename the name to something more obvious. I was scared to change it as I didn't know if it did anything.

```suggestion
   name: Automatic Publication 
```

File: docs/index.html [hunk 5]
```diff
@@ -26,7 +26,14 @@
 			function fixLinks() {
 				document.querySelectorAll("section a").forEach(
 					/** @param {HTMLAnchorElement} a */ a => {
-						if (a.host === location.host && a.pathname.includes("options.md")) {
+						const path = a.pathname;
```
Comment (marcoscaceres) [inline docs/index.html]: I wonder if we should switch this to spaces... tabs looks very weird... almost makes JS look like C++!

Comment (marcoscaceres) [reply docs/index.html]: (followup)

Comment (sidvishnoi) [reply docs/index.html]: Switch to spaces: 🤮 
Add `?ts=2` in URL to get 2-space indentation in your GitHub 😄

Comment (marcoscaceres) [reply docs/index.html]: 🤺

**Review by sidvishnoi — APPROVED** [review]

**Pull request merged**